sherpa/astro/ui/utils.py

```diff
@@ -1841,14 +1870,35 @@ def load_pha(self, id, arg=None, use_errors=False):
 
         phasets = self.unpack_pha(arg, use_errors)
 
-        if numpy.iterable(phasets):
-            num = len(phasets)
-            for id, pha in enumerate(phasets):
-                self.set_data(id + 1, pha)
-            if num > 1:
-                info("Multiple data sets have been input: 1-%s" % num)
-        else:
+        if not numpy.iterable(phasets):
             self.set_data(id, phasets)
+            return
+
+        # One issue with the following is that if there's
+        # only one file and id is a string then the
```

Comment:
You mean "one dataset in the file" and not "one file", right?

Reply:
I'm updating the comment to be clearer.